File: public/javaPrograms/firstJava/Four.java
 
package javaPrograms.firstJava;
import java.util.*;
public class Four {
public static void main(String[] args) {
Scanner s = new Scanner(System.in);
while(true){
    System.out.print("Enter two no's :");
    int n1 = s.nextInt();
    int n2 = s.nextInt();
    System.out.print("set operator :");
    char op =s.next().charAt(0);
    if(op == '+'){
        System.out.println(n1+""+op+""+n2+"="+(float)(n1+n2));
    }
    else if(op == '-'){
        System.out.println(n1+""+op+""+n2+"="+(float)(n1-n2));
    }
    else if(op == '*'){
        System.out.println(n1+""+op+""+n2+"="+(float)(n1*n2));
    }
    else if(op == '/'){
        System.out.println(n1+""+op+""+n2+"="+(float)(n1/n2));
    }
    else{
        System.out.println("operator invalid");
    }
}
     
        
}
}

File: src/programs/functions/four.js
import { useState,useEffect } from "react"
import FirstJava from "../../topics/FirstJava"
import FirstJavaFour from "./four"
import FunctionsFive from "./five"
import Functions from "../../topics/Functions"
const FunctionsFour = ({qn,backPage,qn1,qn2}) =>{
    const q5 = "5.Define a method that returns the product of two numbers entered by user."
    const [file,setFile] = useState("Loading Java FIle...")
    const url = window.location.href
    const back = (page) =>{
        backPage(page)
    }
    useEffect(()=>{
    async function fetchFile() {
        try{
    const res = await fetch(`${url}javaPrograms/functions/Four.java`)

    const cont = await res.text();
    
    setFile(cont)
    }
    catch(err){
    console.log(err);
    }

}
 
fetchFile() 
},[])
    return(<div className="program-body"><div className='topics-logo'>{qn}</div>
        <div className='program-content'>


<div className='program-hint program'><pre>{file}</pre></div>

<div className="program-btns">
<div onClick={() => back(<Functions/>)} className='program-btn'>Go To Map</div>
<div onClick={() => back(<FunctionsFive qn = {q5} backPage={backPage} />)} className='program-btn'>Next</div>
</div>

</div></div>)
}
export default FunctionsFour